gocourse02/zoo/lion.go

```diff
@@ -0,0 +1,33 @@
+package zoo
+
+import (
+	"fmt"
+
+	"gocourse02/utils"
+)
+
+type Lion struct {
+	*Animal
+	color string
+}
+
+func (l *Lion) Color() string {
+	return l.color
+}
+
+func NewLion(animal *Animal, color string) *Lion {
+	return &Lion{
+		Animal: animal,
+		color:  color,
+	}
+}
+
+func (l *Lion) Multiply(otherLion Lion) *Lion {
```

Comment:
Можна скоротити назву параметра без втрати читабельності:
```suggestion
func (l *Lion) Multiply(lion Lion) *Lion {
```